submissions/Levi123/domjs/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>JS DOM</title>
+    <link rel="stylesheet" href="src/css/style.css">
+</head>
+<body>
+    <!--HEADER-->
+    <section class="header-navigation">
+        <div class="wrapper header-navigation__wrapper">
+            <a 
+                href="#" 
+                class="header-navigation_link"
+            >
+                <img 
+                    src="src/assets/logo.svg" 
+                    alt="logo"
+                >
+            </a>
+        </div>
+    </section>
+    <!--NAVIGATION AND ABOUT PRODUCT-->
+    <section class = "about-product">
+        <div class="wrapper about-product__wrapper">
+            <div class="about-product__navigation">
+                <button 
+                    href="#" 
+                    class="about-product_navigation-menu button" id="1">1 series
+                </button>
+                <button
+                    href="#" 
+                    class="about-product_navigation-menu button" id="2">2 series
+                </button>
+                <button
+                    href="#" 
+                    class="about-product_navigation-menu button" id="3">3 series
+                </button>
+                <button
+                    href="#" 
+                    class="about-product_navigation-menu button" id="4">4 series
+                </button>
+                <button
+                    href="#" 
+                    class="about-product_navigation-menu button" id="5">5 series
+                </button>
```

Comment:
It is better to use the dataset properties.

Reply:
> Краще використовувати властивості набору даних.

don't understand you

Reply:
Read about [data attributes](https://developer.mozilla.org/ru/docs/Learn/HTML/Howto/Use_data_attributes)

Reply:
> Read about [data attributes](https://developer.mozilla.org/ru/docs/Learn/HTML/Howto/Use_data_attributes)

done